tests/python/pytorch/graphbolt/test_subgraph_sampler.py

```diff
@@ -970,7 +966,7 @@ def test_SubgraphSampler_unique_csc_format_Homo_Node_gpu(labor):
         deduplicate=True,
     )
 
-    if torch.cuda.get_device_capability()[0] < 7:
+    if F.ctx() != F.cpu() and torch.cuda.get_device_capability()[0] < 7:
```

Comment:
I'd prefer to keep the check on the top of the test cases. CPU is actually not running even after your fix.